extras/userdoc/md/documentation/connection-management.md

```diff
@@ -442,17 +437,15 @@ Example new connection routine:
     syn_dict = {'model': 'my_synapse', 'weight': w0, 'delay': w0}
     nest.Connect(A, B, conn_dict, syn_dict)
 
-Topological Connections
------------------------
+## Topological Connections
 
 If the connect functions above are not sufficient, the topology provides more
 sophisticated functions. For example, it is possible to create receptive field
 structures and much more! See
 [Topological Connections](Topology_UserManual.pdf)
```

Comment:
No it does not. I changed it. 
I need to find the right place for the pdf and other linked files.